src/NuGet.Core/NuGet.CommandLine.XPlat/ListPackage/ListPackageOutputContent.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.CommandLine.XPlat.ListPackage
+{
+    /// <summary>
+    /// List package json output
+    /// </summary>
+    internal class ListPackageOutputContent
```

Comment:
Do we absolutely need this POCO class? 

I see the renderers themselves create it just to pass it to local methods. That can just as easily be 3 parameters.

Reply:
Yes, we still need this need to pass data to JsonSerializer.

https://github.com/NuGet/NuGet.Client/blob/4ce40e89eacc0e43f1391b1f9bc9c9a20689c7cd/src/NuGet.Core/NuGet.CommandLine.XPlat/ListPackage/ListPackageJsonRenderer.cs#L372-L394

Reply:
@nkolev92 
Now passing all the tests, ready for review.

Reply:
I'm not sure I understand your point. 

You're not really using the type for any data passing. You just have the type so that you have 1 argument instead of 3.

Reply:
Problem was `JsonSerializer` doesn't let me use use instance fields of `ListPackageJsonRenderer`, we have to pass new object  or use static variables here. 
https://github.com/NuGet/NuGet.Client/blob/4ce40e89eacc0e43f1391b1f9bc9c9a20689c7cd/src/NuGet.Core/NuGet.CommandLine.XPlat/ListPackage/ListPackageJsonRenderer.cs#L374

Reply:
OK, I see it now. 

You're using a converter, and in the converter you actually manually write the values. 
I find it a bit unusual that you have both a converter + manual json writing. Normally I'd say you use one. 

Given you are already writing manually, you don't need to use a converter, you write directly to the json text writer. 

Either way, I don't care too much for now. If others provide the same feedback then you can change it. 
I'll leave it up to you.

Reply:
You're right, I made mistake. I'll remove the `Converter` now.

Reply:
Deleted this  file.